tests/incremental/00-basic/03-changed_start_state2.patch

```diff
@@ -0,0 +1,19 @@
+--- a/tests/incremental/00-basic/03-changed_start_state2.c
++++ b/tests/incremental/00-basic/03-changed_start_state2.c
+@@ -1,13 +1,13 @@
+ #include <assert.h>
+
+-int g = 1;
++int g = 2;
+
+ int main() {
+     // After the presolve phase, g is in the start state but neither in the context nor in the start variables.
+     // If the change of the start state of main would not be propagated by the call to side on all start variables, the
+     // asserts in the incremental run would wrongly fail. Side however only joins with the previous value instead of
+     // overwriting, therefore the current imprecision.
+-    assert(g == 1);
+-    assert(g != 2);
++    assert(g != 1); // TODO (restarting)
++    assert(g == 2); // TODO
```

Comment:
Why are these TODO now? In the previous copy of this test they were not specified as unknown, neither before nor after.

Reply:
With commit 2d99ffeff615e49663a7c6724398bac47cd51ea1 we changed the handling of start functions whose start state changed without the context being different. We now propagate the changes in the start state by calling side on all start variables. This joins with the previous value instead of overwriting. Due to this, the assertions in the test cases `changed_start_state1` and `changed_start_state2` could not be verified anymore because the abstraction is more imprecise after the change mentioned above. I think the asserts should be provable when one can restart globals, therefore I have used the `TODO`. But I will gladly change this if it is unfitting and you can suggest something else. I also thought about whether these test cases should be removed instead, because with the `UNKNOWN` assert in the incremental run they don't seem very meaningful.